src/app/header/header.component.ts

```diff
@@ -1,10 +1,20 @@
 import { Component } from '@angular/core';
 
 @Component({
-  selector: 'app-header',
+  selector: 'po-header',
   templateUrl: './header.component.html',
   styleUrls: ['./header.component.css']
 })
 export class HeaderComponent {
-
+  menuButton: string = "hamburger menu";
+  displayMenu: boolean = false;
+  constructor() { }
+  menuList(clicked: boolean) {
```

Comment:
Change this method name to `showMenu()`. `menuList` sounds like you are fetching menus here.